src/DotNetWorker/Context/Features/DefaultModelBindingFeature.cs

```diff
@@ -54,7 +54,10 @@ public DefaultModelBindingFeature(IEnumerable<IConverter> converters)
                 if (TryConvert(converterContext, out object? target))
                 {
                     _parameterValues[i] = target;
-                    continue;
+                }
+                else if (source is not null)
+                {
+                    throw new InvalidCastException($"Cannot convert input parameter '{param.Name}' for Function '{context.FunctionDefinition.Name}' to type '{param.Type.FullName}' from type '{source.GetType().FullName}'.");
```

Comment:
Per feedback:
- Iterate through all parameters and collect error messages - throw as one
- Introduce model binding exception